tests/scripts/elasticsearchstore_integration_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#get directory of current file
+DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
+
+echo "Beginning Elasticsearchstore integration tests"
+
+echo "Creating venv"
+rm -rf elasticsearchstore_test_env
+python3.6 -m venv elasticsearchstore_test_env
+. elasticsearchstore_test_env/bin/activate
+pip install -U pip setuptools
+pip install -e .
+pip install -r tests-requirements.txt
```

Comment:
You don't need to do it here, you can do it directly in github actions. (check the 'Set up python 3.6' and 'Install dependencies' parts)